shared/lib_battery_voltage.cpp

```diff
@@ -428,20 +452,20 @@ double voltage_dynamic_t::calculate_current_for_target_w(double P_watts, double
 
     newton<double, std::function<void(const double *, double *)>, 1>(x, resid, check, f,
                                                                      100, 1e-6, 1e-6, 0.7);
+
     return x[0] * params->num_strings * direction;
 }
 
 void voltage_dynamic_t::solve_current_for_charge_power(const double *x, double *f) {
     double I = x[0];
-    double V = _E0 - _K * solver_Q / (solver_q + I * params->dt_hr) +
+    double V = _E0 - _K * solver_Q_mod / (solver_Q_mod - (solver_Q - (solver_q + I * params->dt_hr))) +
```

Comment:
I'm mostly curious, why isn't this `_K * solver_Q_mod / (solver_q + I * params->dt_hr)`? Additionally, why is solver_Q in the exponent unchanged?